pgmpy/tests/test_models/test_BayesianNetwork.py

```diff
@@ -1807,23 +1805,92 @@ def test_missing_scheme_sample(self):
         samples = self.con_model.simulate(n_samples=1000)
         self.assertFalse(samples.isnull().values.any())
 
-        samples = self.con_model.simulate(
-            n_samples=1000, include_missing=True, missing_prob=0.9
+        cpd = TabularCPD(
+            "Z*",
+            2,
+            [[0.5], [0.5]],
```

Comment:
I would suggest using values other than 0.5 everywhere because we would end up with 0.5 probability of missingness if the missingness is random and doesn't use the specified CPD.